packages/vike-react/src/renderer/styles.css

```diff
@@ -0,0 +1,17 @@
+.vike-react-apollo-loading-component {
```

Comment:
How about we define this as an inline style?

The neat thing would also be that it doesn't get loaded if the user overrides `+LoadingComponent`.